.circleci/config.yml

```diff
@@ -127,18 +120,42 @@ workflows:
               ignore:
                 - master
 
-      - test:
+      # UNIX tests
+      - test-unix:
+          name: Unix Tests for Node << matrix.node_version >>
+          context: nodejs-install
+          requires:
+            - lint
+          matrix:
+            parameters:
+              node_version: ["14.17.6", "12.0.0", "10.21.0"]
+          filters:
+            branches:
+              ignore:
+                - master
+
+      # Windows tests
+      - test-windows:
+          name: Windows Tests for Node << matrix.node_version >>
+          context: nodejs-install
+          requires:
+            - lint
           matrix:
             parameters:
-              os: [linux, macos, windows]
-              node_version: ["10.22", "12.18", "14.8"]
+              node_version: ["14.17.6", "12.0.0", "10.21.0"]
+          filters:
+            branches:
+              ignore:
+                - master
 
       - release:
           name: Release
           context: nodejs-lib-release
+          node_version: "lts"
```

Comment:
Great work here @ekbsnyk! 🎉 
Just one question, why did you decide to go with `lts` as release version?

Reply:
Just going with the same release version as we previously used, as far as I could tell (as per https://github.com/snyk/snyk-cpp-plugin/blob/master/.circleci/config.yml#L16-L17)

Reply:
```
executors:
  docker-node:
    parameters:
      version:
        default: "lts"
        type: string
    docker:
      - image: cimg/node:<<parameters.version>>
```
defines the executor used in the release job:
```
  release:
    resource_class: small
    executor:
      name: docker-node
    steps:
      - checkout
      - npmrc
      - install_deps
      - run:
          name: Release
          command: npx semantic-release
```

Reply:
Cool beans 🎉